packages/cuba-react/src/app/Security.ts

```diff
@@ -24,17 +29,27 @@ export class Security {
   };
 
   @action
-  loadPermissions() {
+  async loadPermissions() {
```

Comment:
https://mobx.js.org/best/actions.html#async-await
when using `async` only statements before first await call will be batched in a single action